### PR TITLE
Add navigation and Hero section

### PR DESCRIPTION
**Pull request by emmyobonyo**
### Tasks completed.
- Add navigation.
- Add Hero section images.
- Add Hero heading and Paragraph.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/.eslintrc.json
```
Comment (KnightWorker37) [inline .gitignore:25]: Lint configurations should be kept in git so everyone follows the same style

File: public/robots.txt [hunk 2]
```diff
@@ -1,3 +0,0 @@
-# https://www.robotstxt.org/robotstxt.html
-User-agent: *
```
Comment (KnightWorker37) [inline public/robots.txt:2]: It's not necessary to delete this file. Read more about what it's doing here: https://developer.mozilla.org/en-US/docs/Glossary/Robots.txt

File: src/App.test.js [hunk 3]
```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react';
```
Comment (KnightWorker37) [inline src/App.test.js:1]: I hope there will be tests in future for this app?

File: src/index.js [hunk 4]
```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
```
Comment (KnightWorker37) [inline src/index.js:5]: Hope you deleted after understanding what you are missing out on? (https://create-react-app.dev/docs/measuring-performance/)

File: src/setupTests.js [hunk 5]
```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
```
Comment (KnightWorker37) [inline src/setupTests.js:5]: I'm hoping for tests in the future?

**Review by KnightWorker37 — APPROVED** [review]
Not blocking issues but would be nice if you reply them.

Also I see you deleted a lot of assets but was `manifest.json` updated accordingly?

File: .gitignore (hunk 1 above)
Comment (emmyobonyo) [reply .gitignore:25]: Noted

File: public/robots.txt (hunk 2 above)
Comment (emmyobonyo) [reply public/robots.txt:2]: Noted

File: src/App.test.js (hunk 3 above)
Comment (emmyobonyo) [reply src/App.test.js:1]: Noted

File: src/setupTests.js (hunk 5 above)
Comment (emmyobonyo) [reply src/setupTests.js:5]: Agreed. The instructor isn't doing tests but I'll definitely do it in the future.

**Pull request merged**